### ISSUE TITLE
Add ability to instantiate front-end application, like running .net apps within front-end apps

### ISSUE THREAD

**Issue by Artur2** [dotnet/tye#665]
### What should we add or change to make your life better?
Add ability to run command (like starting npm application)
### Why is this important to you?
I'm working on backend microservices and frontend angular app

**Comment by jkotalik** [comment]
Yeah this is one of our top priorities and will be one of the next features we add to tye.
